src/pkcs1_5.c

```diff
@@ -244,23 +244,25 @@ int ri_manifest_sign_v1_5(struct image *image)
 {
 	struct fw_image_manifest_v1_5 *man = image->fw_image;
 
-	pkcs_v1_5_sign_man_v1_5(image, man,
-				(void *)man + MAN_CSS_MAN_SIZE_V1_5,
-				image->image_end - sizeof(*man));
-	return 0;
+	char *const data1 = (char *)man + MAN_CSS_MAN_SIZE_V1_5;
+	unsigned const size1 = image->image_end - sizeof(*man);
+
+	return pkcs_v1_5_sign_man_v1_5(image, man, data1, size1);
 }
 
 int ri_manifest_sign_v1_8(struct image *image)
 {
 	struct fw_image_manifest_v1_8 *man = image->fw_image;
 
-	pkcs_v1_5_sign_man_v1_8(image, man, (void *)man + MAN_CSS_HDR_OFFSET,
-				sizeof(struct css_header_v1_8) -
-				(MAN_RSA_KEY_MODULUS_LEN +
-				 MAN_RSA_KEY_EXPONENT_LEN +
-				 MAN_RSA_SIGNATURE_LEN),
-				(void *)man + MAN_SIG_PKG_OFFSET_V1_8,
-				(man->css.size - man->css.header_len)
-				 * sizeof(uint32_t));
-	return 0;
+	char *const data1 = (char *)man + MAN_CSS_HDR_OFFSET;
+	unsigned const size1 =
```

Comment:
`uint32_t` would be shorter and consistent with the rest of a code

Reply:
I have no strong opinion on `uint32_t` versus `unsigned` here but did you see that  `uint32_t` would be _inconsistent_ with the functions this is passed to? I used `unsigned` here because the only purpose of the constant (optimized away by the compiler) is to give a name to the function parameter so it made sense to me to also give the reader the type of the function parameter at the same time.

BTW this line already involves 2 integer types: `size_t` and `unsigned`. Using `uint32_t` would add a third one.